Sources/MockoloFramework/Templates/ClassTemplate.swift

```diff
@@ -114,22 +125,21 @@ private func extraInitsIfNeeded(_ entities: [(String, Model)],
         }
         .joined(separator: ", ")
         
-        paramsAssign = extraInitParamCandidates.map { p in
-            return """
-            self.\(p.name) = \(p.name.safeName)
-            """
+        paramsAssign = initParamCandidates.map { p in
+            return "\(String.spaces8)self.\(p.name) = \(p.name.safeName)"
+            
         }.joined(separator: "\n")
         
+        
         initTemplate = """
-        \(accessControlLevelDescription)init(\(params)) {
-            \(paramsAssign)
-            \(String.doneInit) = true
-        }
+        \(String.spaces4)\(accessControlLevelDescription)init(\(params)) {
```

Comment:
nit: I would write an extension on String that indents by N. Or something like `4.spaces` or `1.tabs`.

Reply:
oh right, how about 4.ish ?